### PR TITLE
Adding new module azure_rm_publicipaddress

### PR DESCRIPTION
**Pull request by chouseknecht**
##### ISSUE TYPE
- New Module Pull Request
##### COMPONENT NAME

azure_rm_publicipaddress.py
##### ANSIBLE VERSION

```
ansible 2.1.0 (windows_azure 7f3ea66c2b) last updated 2016/04/18 18:17:11 (GMT -400)
  lib/ansible/modules/core: (detached HEAD c52f475c64) last updated 2016/04/10 08:32:14 (GMT -400)
  lib/ansible/modules/extras: (detached HEAD 5abb914315) last updated 2016/04/10 08:23:55 (GMT -400)
  config file =
  configured module search path = Default w/o overrides
```
##### SUMMARY

New module for managing public IP addresses on the Azure cloud.

### CODE REVIEW

**Comment by gregdek** [comment]
Thanks @chouseknecht for this new module. When this module receives 'shipit' comments from two community members and any 'needs_revision' comments have been resolved, we will mark for inclusion.

[This message brought to you by your friendly Ansibull-bot.]

File: cloud/azure/azure_rm_publicipaddress.py [hunk 1]
```diff
@@ -0,0 +1,308 @@
...
+            - Valid azure location. Defaults to location of the resource group.
+        default: resource_group location
+        required: false
+    tags:
+        description:
+            - "Dictionary of string:string pairs to assign as metadata to the object. Metadata tags on the object
+              will be updated with any provided values. To remove tags use the purge_tags option."
+        required: false
+        default: null
+    purge_tags:
```
Comment (bcoca) [inline cloud/azure/azure_rm_publicipaddress.py:80]: I better option would be one that toggles between appending tags or making it a canonical list. I know this follows the aws modules, but that was a bad name there too.

File: cloud/azure/azure_rm_publicipaddress.py [hunk 2]
```diff
@@ -0,0 +1,308 @@
...
+
+    - name: Delete public ip
+      azure_rm_publicipaddress:
+        resource_group: testing
+        name: my_public_ip
+        state: absent
+'''
+
+RETURN = '''
+changed:
```
Comment (bcoca) [inline cloud/azure/azure_rm_publicipaddress.py:111]: again, changed should not be specified here as it is part of 'default' returns

File: cloud/azure/azure_rm_publicipaddress.py (hunk 1 above)
Comment (chouseknecht) [inline cloud/azure/azure_rm_publicipaddress.py:80]: Not changing this.

**Pull request merged**